scripts/compare_indexes/compare_indexes.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include "index.h"
+
+int main(int argc, char* argv[]) {
+	Index index1;
+	Index index2;
+	index1.load(argv[1]);
```

Comment:
To prevent this type of "two-stage" `Index` creation it would probably be more convenient to have a static method on the `Index` class called `from_file` which returns a new instance of `Index` (or an alternate constructor that takes a `fs::path`). That way the use of this would also make it clearer as to what's happening in the code. 
i.e
```c++
auto index1 { Index::from_file(argv[1]) }
```

Reply:
I guess this should be done in several places also, thus I created an issue addressing this: https://github.com/rmcolq/pandora/issues/144